### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by kyleskom** [OpenBB-finance/OpenBB#636] [General]
Just wanted to say this project is awesome and really well done. Just wanted to say thanks!

**Comment by DidierRLopes** [comment]
> Just wanted to say this project is awesome and really well done. Just wanted to say thanks!

Thank you @kyleskom, we appreciate your support! The terminal is still in his infancy, there's still a lot more to come!

If you want to be part of this, just join our discord and let's chat there :)
